codex/node.nim

```diff
@@ -675,32 +671,45 @@ proc onStore(
 
     return success()
 
-  without indexer =?
-    manifest.verifiableStrategy.init(0, manifest.blocksCount - 1, manifest.numSlots).catch,
-    err:
-    trace "Unable to create indexing strategy from protected manifest", err = err.msg
-    return failure(err)
-
   if slotIdx > int.high.uint64:
```

Comment:
Could this ever happen - I don't think it can? It would die during conversion since you can't simply fit a larger number than what the type could handle.

Reply:
ah, we're converting down to an int.